docs/scenarios_authoring/RRDataSourcesAndSinks.md

```diff
@@ -37,7 +37,26 @@ hint field names and will perform additional validations on SpEL expressions inv
 
 The `response` sink configuration form will show a list of fields defined in [Input schema](#request-response-schema).
 
-![RR sink](img/rrSink.png "Kafka sink")
+![RR sink](img/rrSink.png "RR sink")
 
 If you prefer to define the whole response as one expression containing value of the HTTP response instead of filling
 separate autogenerated fields, you can do that by switching `Raw editor` to `true`.
+
+When using the Raw editor, you can choose the validation mode:
+* strict - requires providing all fields, including optional fields, requires exact matching;
+* lax - requires providing only required fields, you can skip optional fields, exact matching is not required;
```

Comment:
lets have definition of strict and lax in one place. It should be the same for Kafka and also RR, right?

Reply:
I've discussed this with @mproch and we decided to duplicate it.

Reply:
We changed our minds :) And it was moved to one place as you have suggested.